src/cart/crt_hg.c

```diff
@@ -19,42 +19,43 @@ struct crt_na_dict crt_na_dict[] = {
 	{
 		.nad_type	= CRT_NA_SM,
 		.nad_str	= "sm",
+		.nad_alt_str	= "",
```

Comment:
It would probably be better to allow this to be left blank, to make it safer to add new providers without setting this.  It looks like the only code difference would be on L109 where you'd need to check for NULL.

Reply:
sorry, i missed this comment. I will fix it in a subsequent PR.